client/src/containers/pages/locations/LocationUnitGroup/index.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Table, Input, Popconfirm, Form, Row, Col, Menu, Dropdown, Button, Divider } from 'antd';
+import { MoreOutlined, SearchOutlined, SettingOutlined, PlusOutlined } from '@ant-design/icons';
+// import { Link } from 'react-router-dom';
+// import { getExtraData } from '@onaio/session-reducer';
+import { connect } from 'react-redux';
+// import { Store } from 'redux';
+// import { Dictionary } from '@onaio/utils';
+import '../Location.css';
+import LocationDetail from '../../../../components/locations/LocationDetail';
+
+interface Item {
+  key: string;
+  name: string;
+  status: 'Alive' | 'Not Active';
+  type: string;
+  created: string;
+  lastupdated: string;
```

Comment:
@HaseemUlHaq We should use Date type for these . you can check my branch to see how to handle date type